app/builders/digest_email_builder.rb

```diff
@@ -0,0 +1,65 @@
+class DigestEmailBuilder
+  def initialize(subscriber:, digest_run:, subscription_content_change_results:)
+    @subscriber = subscriber
+    @digest_run = digest_run
+    @results = subscription_content_change_results
+  end
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def call
+    Email.new(
+      subject: subject,
+      body: body,
+      address: subscriber.address
+    )
+  end
+
+  def subject
```

Comment:
I wonder if this and `body` should also be private?

Reply:
Fixed in 299ca1d